### ISSUE TITLE
[TF2] Change contracts from "get x kills in a single life" to "get x kills in a single round"

### ISSUE THREAD

**Issue by ShadowMan44** [ValveSoftware/Source-1-Games#4419]
Hey! Has this ever happened to you:

You're on your shotgun contract right, you get one lucky kill with your shotgun, you're doing good running around the map to get your next one, then **oops** you get backstabbed by an unsuspecting Spy, or you end up losing the round because it took too long for you to find the right moment to get your kill in.

I propose this; Instead of needing to get 3 kills with a shotgun in a single life, how about just the whole _round_ instead?

"But 44, wouldn't that make it too easy?". Well, in some cases it would be a lot eas_ier_ yes but generally speaking it's really hard to get kills with secondary weapons, you don't normally try to use them as your main weapon.

"But 44, can't you just get a friendly to help you?". Yes that's true, but people will cheese their contracts regardless of what it is to get it done quicker, why would this be any different?

"But 44, how would this affect how the points are given?" Simple, just give the player a point whenever they get a kill at any point in the round.

Look this might sound like a "skill issue" but read the above paragraph and tell me getting killed by an invisible spy, which was totally  out of your control _your_ fault?

**Comment by Ismaciodismorphus** [comment]
Yes getting killed by a spy is your fault as the spy is never invisible when he kills you and you have the simple ability to turn around at any time and shoot sus teammates

**Comment by ShadowMan44** [comment]
> Yes getting killed by a spy is your fault as the spy is never invisible when he kills you and you have the simple ability to turn around at any time and shoot sus teammates

See everybody always says "oh, just turn around"
By the time you turn around the Spy has already had enough time to stab you in the back so what's the point of doing it if you're not a freaking Pyro.

**Comment by MikhailaTF** [comment]
You could argue that depending on a player's skill level that nearly all your deaths are either totally within or completely outside of your own control. You get backstabbed less as you learn to spycheck certain spots on certain maps, for example.
Regardless, I feel like these contracts were made to provide a little bit of a fun challenge and perhaps to get people to experiment with new weapons and playstyles. You could wonder what the point of having them in the first place is if anyone can complete them on their first try.

**Comment by rabscootle** [comment]
I see no need to change the contract. It's a _bonus_ objective (meaning it won't block anyone from progressing further in the campaign aside from 100% completion). The game already indicates that these are supposed to be more difficult.

**Comment by ShadowMan44** [comment]
> I see no need to change the contract. It's a _bonus_ objective (meaning it won't block anyone from progressing further in the campaign aside from 100% completion). The game already indicates that these are supposed to be more difficult.

The Pyroland contracts requires you to get _all 3_ objectives on the contract in order to progress to the Hot Hand contract. Because it requires _9_ stars, and the others only require 1, meaning the only way to get to it is to complete the bonus contracts which **do** require needing to get multiple kills in 1 life.
